pkgs/native_assets_builder/test_data/add_asset_link/hook/link.dart

```diff
@@ -7,17 +7,15 @@ import 'package:native_assets_cli/native_assets_cli.dart';
 void main(List<String> arguments) async {
   await link(arguments, (config, output) async {
     final builtDylib = config.assets.first as NativeCodeAsset;
-    output
-      ..addAsset(
-        NativeCodeAsset(
-          package: 'add_asset_link',
-          name: 'dylib_add_link',
-          linkMode: builtDylib.linkMode,
-          os: builtDylib.os,
-          architecture: builtDylib.architecture,
-          file: builtDylib.file,
-        ),
-      )
-      ..addDependency(config.packageRoot.resolve('hook/link.dart'));
```

Comment:
Why remove dependencies in link hooks?

Reply:
The Dart files don't need to be in the dependencies anymore.

Reply:
Forgot about that PR..